app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/settings/SettingsScreen.kt

```diff
@@ -60,7 +76,35 @@ fun SettingsScreenWrapper(
 
 @Composable
 private fun SettingsScreen(onItemClick: (Any) -> Unit) {
-    Text("Settings screen")
+    Column(modifier = Modifier.verticalScroll(rememberScrollState())) {
+        Text(
```

Comment:
This is just for the poster example, but later it will be a toolbar, right?

Reply:
Yes, but I decided to not bother with adding the toolbar and the scaffold for now